synapse/handlers/oidc.py

```diff
@@ -246,6 +253,78 @@ async def handle_oidc_callback(self, request: SynapseRequest) -> None:
 
         await oidc_provider.handle_oidc_callback(request, session_data, code)
 
+    async def handle_backchannel_logout(self, request: SynapseRequest) -> None:
+        """Handle an incoming request to /_synapse/client/oidc/backchannel_logout
+
+        This extracts the logout_token from the request and try to figure out
+        from whom it is coming from. This works by matching the iss claim with
+        the issuer and the aud claim with the client_id.
```

Comment:
I think this is a little more readable?

```suggestion
        This extracts the logout_token from the request and tries to figure out
        which account it is associated with. This works by matching the iss claim
        with the issuer and the aud claim with the client_id.
```

Reply:
Ha, I did not properly read your suggestion. It's not "which account it is associated with" but rather "from which IdP it is coming from"

Reply:
Isn't this matching to a particular user though, not just the IdP?